<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trainer</class>
 <widget class="QWidget" name="Trainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="3">
    <widget class="QSpinBox" name="trnWeight_3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="minimum">
      <number>-125</number>
     </property>
     <property name="maximum">
      <number>125</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_36">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_37">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_38">
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="trnLabel_1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Rud</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="trnMode_1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Beeper volume

0 - Quiet.  No beeps at all.
1 - No Keys.  Normal beeps but menu keys do not beep.
2 - Normal.
3 - Loud.
4 - Extra loud.</string>
     </property>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>+= (Sum)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:= (Replace)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="trnChn_1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Beeper volume

0 - Quiet.  No beeps at all.
1 - No Keys.  Normal beeps but menu keys do not beep.
2 - Normal.
3 - Loud.
4 - Extra loud.</string>
     </property>
     <item>
      <property name="text">
       <string>chn1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="trnWeight_1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="minimum">
      <number>-125</number>
     </property>
     <property name="maximum">
      <number>125</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="trnLabel_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ele</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="trnMode_2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Beeper volume

0 - Quiet.  No beeps at all.
1 - No Keys.  Normal beeps but menu keys do not beep.
2 - Normal.
3 - Loud.
4 - Extra loud.</string>
     </property>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>+= (Sum)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:= (Replace)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="trnChn_2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Beeper volume

0 - Quiet.  No beeps at all.
1 - No Keys.  Normal beeps but menu keys do not beep.
2 - Normal.
3 - Loud.
4 - Extra loud.</string>
     </property>
     <item>
      <property name="text">
       <string>chn1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="trnChn_3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Beeper volume

0 - Quiet.  No beeps at all.
1 - No Keys.  Normal beeps but menu keys do not beep.
2 - Normal.
3 - Loud.
4 - Extra loud.</string>
     </property>
     <item>
      <property name="text">
       <string>chn1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="trnWeight_2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="minimum">
      <number>-125</number>
     </property>
     <property name="maximum">
      <number>125</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="trnLabel_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Thr</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="trnMode_3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Beeper volume

0 - Quiet.  No beeps at all.
1 - No Keys.  Normal beeps but menu keys do not beep.
2 - Normal.
3 - Loud.
4 - Extra loud.</string>
     </property>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>+= (Sum)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:= (Replace)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="trnLabel_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ail</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="trnMode_4">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Beeper volume

0 - Quiet.  No beeps at all.
1 - No Keys.  Normal beeps but menu keys do not beep.
2 - Normal.
3 - Loud.
4 - Extra loud.</string>
     </property>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>+= (Sum)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:= (Replace)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="trnChn_4">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Beeper volume

0 - Quiet.  No beeps at all.
1 - No Keys.  Normal beeps but menu keys do not beep.
2 - Normal.
3 - Loud.
4 - Extra loud.</string>
     </property>
     <item>
      <property name="text">
       <string>chn1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>chn4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="trnWeight_4">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="minimum">
      <number>-125</number>
     </property>
     <property name="maximum">
      <number>125</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
